### ISSUE TITLE
Change the number of hidden layers

### ISSUE THREAD

**Issue by sadamov** [MeteoSwiss/neural-lam#8]
The latent space representation in the graph can affect the way the model is learning/generalizing the training data. In this issue it is suggested to change the number of hidden layers (in the small MLPs along edges and nodes) and observe whether this can improve the training. 

Suggestion:

1. change the hidden_dim, by setting this argument in the `train_model.py` call in the `slurm_train.sh` script
```
parser.add_argument('--hidden_layers', type=int, default=1,
                        help='Number of hidden layers in all MLPs (default: 1)')
```
A good first try would be to change it from 1 -> 2.
3. start a training with 40 epochs and validation every 20 epochs
4. compare loss curves to previous model runs on wandb
